test/driver.spec.js

```diff
@@ -0,0 +1,250 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import chai from 'chai';
+import {Ed25519VerificationKey2020} from
+  '@digitalbazaar/ed25519-verification-key-2020';
+import {driver} from '../lib/index.js';
+
+chai.should();
+const {expect} = chai;
+
+describe('did:jwk method driver', () => {
+  describe('get', () => {
+    it('should get the DID Document for a EdDSA did:jwk DID', async () => {
+      const didJwkDriver = driver();
+
+      const did = 'did:jwk:eyJraWQiOiJ1cm46aWV0ZjpwYXJhbXM6b2F1dGg6andrLXRodW1icHJpbnQ6c2hhLTI1NjpraXJBWXVfNWNoUEJhbEJLNFk3dTJhTGtPUzRLYXVQNkFWQjE4NXRYbHdJIiwia3R5IjoiT0tQIiwiY3J2IjoiRWQyNTUxOSIsImFsZyI6IkVkRFNBIiwieCI6ImlQaEFZS3FJUEs5cm5nX3VlZGhwWHpJQzJ2T01uOFZ0R29vaGRuQVZsckEifQ';
+      const keyIdFragment = '#0';
+      const keyId = `${did}${keyIdFragment}`;
+      const keyIdCandidates = [keyId, keyIdFragment];
+      const didDocument = await didJwkDriver.get({did});
+
+      expect(didDocument.id).to.equal(did);
+      expect(didDocument['@context']).to.eql([
+        'https://www.w3.org/ns/did/v1',
+        'https://w3id.org/security/suites/jws-2020/v1'
```

Comment:
jws-2020 has been abandoned. The new context for a JsonWebKey is here: https://w3id.org/security/jwk/v1

It doesn't have a date in its type anymore either, just `JsonWebKey`. You'll want to take a look at the context.

Reply:
I have opened #2 for this issue